jetstream/workflows/run.yaml

```diff
@@ -72,12 +72,15 @@ spec:
     container:
       image: gcr.io/moz-fx-data-experiments/jetstream:latest
       command: [
-        jetstream, --log_to_bigquery, run, 
-        "--date={{inputs.parameters.date}}", 
-        "--experiment_slug={{inputs.parameters.slug}}", 
-        "--dataset_id={{workflow.parameters.dataset_id}}", 
-        "--project_id={{workflow.parameters.project_id}}",
-        "--bucket={{workflow.parameters.bucket}}"
+        jetstream, --log_to_bigquery, run
+      ]
+      args: [ 
+        "--date={{inputs.parameters.date}} 
+        --experiment_slug={{inputs.parameters.slug}} 
+        --dataset_id={{workflow.parameters.dataset_id}} 
+        --project_id={{workflow.parameters.project_id}} 
+        --bucket={{workflow.parameters.bucket}} 
+        {{workfow.parameters.analysis_periods}}"
```

Comment:
Argo doesn't seem to be able to handle whitespace-separated arguments as part of `command`, so using `args` instead

Reply:
mhhh this is throwing an error `ValueError: unconverted data remains`

Reply:
`oh {{workfow.parameters.analysis_periods}}` doesn't actually get replaced, I'll push a fix